### DISCUSSION TITLE
How to design the first day of every month?

### DISCUSSION THREAD

**Discussion by liuqijie6** [agronholm/apscheduler#691] [Q&A] [answered]
Because the time span is too large, it is inconvenient to test.
Does the following expression work for the first day of every month?

```
self.block.add_job(aa, 'cron',  month='1-12', day=1, hour=1, id="aa")
```

**Comment by agronholm** [comment, marked as answer]
Yeah, but you can drop the `month` argument, just like you don't have to explicitly specify `year`, `day` or `day_of_week`.

**Comment by liuqijie6** [comment]
thanks
